Lib/test/test_asyncio/test_base_events.py

```diff
@@ -813,6 +813,22 @@ def create_task(self, coro):
         task._log_destroy_pending = False
         coro.close()
 
+    def test_create_named_task_with_custom_factory(self):
```

Comment:
Please also add `test_create_named_task_with_default_factory`. That would test a bug we fixed in this PR when the default task factory didn't pass the `name=name` to `Task()`.